scripts/tracecontext-integration-test.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+# set -e 
+# hard-coding the git tag to ensure stable builds.
+TRACECONTEXT_GIT_TAG="98f210efd89c63593dce90e2bae0a1bdcb986f51"
+# clone w3c tracecontext tests
+mkdir -p target
+rm -rf ./target/trace-context
+git clone https://github.com/w3c/trace-context ./target/trace-context
+cd ./target/trace-context && git checkout $TRACECONTEXT_GIT_TAG && cd -
+# start example opentelemetry service, which propagates trace-context by 
+# default.
+python ./examples/trace/server.py 1>&2 &
+EXAMPLE_SERVER_PID=$!
+# give the app server a little time to start up. Not adding some sort
+# of delay would cause many of the tracecontext tests to fail being
+# unable to connect.
+sleep 1
+function on-shutdown {
+    # send a sigint, to ensure
+    # it is caught as a KeyboardInterrupt in the
+    # example service.
+    kill -2 $EXAMPLE_SERVER_PID
```

Comment:
This doesn't seem to kill the development server?

Reply:
An alternative way is to run the server, spawn a separate unit test, fetch the test result and shutdown the server. I've explored this [here](https://github.com/w3c/trace-context/blob/f37a97317af5cae528d099c874c8007351e847b5/test/self_test.py#L124) and it seems to work well.
With this approach we don't need to wait for indefinite time and later kill the server process.

Reply:
Yeah, I was trying to find a quick way to do this in bash. I can send a standard kill signal which will kill the server, but at the cost of not calling shutdown. 

In general I'll try to vet this a little more. Thanks @reyang for the example, I'll switch to a python script if I can't get the bash one fixed up more.

Reply:
@c24t for clarification, I've resolved the issue that kept the dev server from being killed (just switched it back to a sigkill). Was there something else you think needed to be improved aside from that?

The approach Reiley linked will not print out the traces as the unit tests enact them, which I found was critical to debugging the issues. The current output intermingles them as shown in https://travis-ci.org/open-telemetry/opentelemetry-python/jobs/604220488#L1083